react-native/react-native.d.ts

```diff
@@ -3354,7 +3386,99 @@ declare namespace  __React {
     }
 
     export interface StyleSheetStatic extends React.ComponentClass<StyleSheetProperties> {
-        create<T>( styles: T ): T;
```

Comment:
@andyshuxin Scratch that, this breaks tuff too. I recommend just leaving the definition as it is right now